operatorconfig/moduleconfig/common/version-values.yaml

```diff
@@ -16,13 +16,10 @@ powerflex:
   # List of Driver versions and modules that supports the version
   v2.3.0:
       authorization: "v1.3.0"
-      replication: "v1.3.0"
```

Comment:
why are these removed?

Reply:
powerflex doesn't support replication yet 
so removing it, i dont know who added it

Reply:
thanks!